language/de/cli.php

```diff
@@ -61,13 +63,28 @@
 	'CLI_DESCRIPTION_GET_CONFIG'				=> 'Gibt den Wert einer Konfigurations-Einstellung aus',
 	'CLI_DESCRIPTION_INCREMENT_CONFIG'			=> 'Erhöht eine Kofigurations-Einstellung',
 	'CLI_DESCRIPTION_LIST_EXTENSIONS'			=> 'Gibt alle in der Datenbank und im Dateisystem vorhandenen Erweiterungen aus.',
+
+	'CLI_DESCRIPTION_OPTION_ENV'				=> 'Name der Umgebung.',
 	'CLI_DESCRIPTION_OPTION_SAFE_MODE'			=> 'Im abgesicherten Modus ausführen (ohne Erweiterungen).',
 	'CLI_DESCRIPTION_OPTION_SHELL'				=> 'Komandozeile starten.',
+
 	'CLI_DESCRIPTION_PURGE_EXTENSION'			=> 'Löscht die angegebene Erweiterung.',
+	'CLI_DESCRIPTION_REPARSER_LIST'				=> 'Listet die Arten von Texten auf, die neu analysiert werden können.',
+	'CLI_DESCRIPTION_REPARSER_REPARSE'			=> 'Analysiert gespeicherte Texte mit dem aktuellen text_formatter-Service neu.',
+	'CLI_DESCRIPTION_REPARSER_REPARSE_ARG_1'	=> 'Text-Art die neu analysiert werden soll. Leer lassen, um alles neu zu analysieren',
+	'CLI_DESCRIPTION_REPARSER_REPARSE_OPT_DRY_RUN'		=> 'Keine Änderungen speichern; nur ausgeben, was passieren würde',
+	'CLI_DESCRIPTION_REPARSER_REPARSE_OPT_RANGE_MIN'	=> 'Niedrigste Eintrags-ID, die verarbeitet wird',
+	'CLI_DESCRIPTION_REPARSER_REPARSE_OPT_RANGE_MAX'	=> 'Höchste Eintrags-ID, die verarbeitet wird',
+	'CLI_DESCRIPTION_REPARSER_REPARSE_OPT_RANGE_SIZE'	=> 'Geschätzte Zahl von Einträgen, die in einem Durchgang verarbeitet werden',
+	'CLI_DESCRIPTION_REPARSER_REPARSE_OPT_RESUME'		=> 'Startet die Analyse dort, wo die letzte Ausführung endete',
```

Comment:
Analysieren ist zwar theoretisch eine Übersetzungsmöglichkeit, aber in meinen Augen passt die Bedeutung hier nicht, weil ich "analysieren" nicht mit Veränderungen an der Datenbank in Verbindung bringen würde, das gleiche gilt für "auswerten". Mit erstellen könnte ich genau wie mit verarbeiten leben.
Am besten passt aber nach wie vor das auch im Duden gelistete [parsen] (http://www.duden.de/rechtschreibung/parsen), denn das schließt alles was notwendig wäre ein.